src/content/docs/installation/installation_t2macbook.mdx

```diff
@@ -3,98 +3,120 @@ title: T2 MacBook Installation
 description: How to install CachyOS on a T2 MacBook
 ---
 
-import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
+import { Steps } from '@astrojs/starlight/components';
 
-## Installation
+CachyOS provides out-of-the-box support for T2 MacBooks, including necessary kernel patches in all provided kernels. However, there are crucial steps required before, during, and after installation for a successful setup, especially regarding Wi-Fi.
 
-### General Information
+:::caution[Potential Risks]
+Completely removing macOS carries risks, like data loss or rendering the device unusable. It's **strongly recommended** to keep a macOS partition for firmware updates and recovery. Refer to the [T2 Linux Roadmap](https://wiki.t2linux.org/roadmap/#can-i-completely-remove-macos) for details.
 
-CachyOS provides out of the box support for the T2 MacBook. The required kernel patches are shipped in all the kernels provided by CachyOS.
+If you still decide to remove macOS, ensure you have a full backup and create a [bootable macOS installer](https://support.apple.com/en-us/101578) beforehand.
+:::
 
-**WiFi connection** is **not available** out of the box because it is a closed source firmware, which CachyOS is not allowed to redistribute.
-The installation needs internet to proceed, so either use a LAN network, or follow [this section](#getting-wifi-firmware-for-the-installation) to get WiFi firmware.
+:::note[Wi-Fi Firmware Required]
+Wi-Fi **will not work** out of the box during or after installation because the required firmware is proprietary and cannot be redistributed by CachyOS. You will need an internet connection (e.g., Ethernet, USB tethering) for the installation process unless you follow the steps below to prepare the Wi-Fi firmware beforehand and enable it during installation.
+:::
 
-### Getting WiFi firmware for the installation
+## Creating a macOS Recovery Installer
 
-<Steps>
-
-1. First boot into macOS and run this command:
-    ```sh
-    curl -sL https://wiki.t2linux.org/tools/firmware.sh | bash -s copy_to_efi
-    ```
-
-2. Now boot into the CachyOS ISO. Open Konsole there, and run these commands:
-    ```sh
-    sudo mkdir -p /tmp/apple-wifi-efi
-    sudo mount /dev/nvme0n1p1 /tmp/apple-wifi-efi
-    bash /tmp/apple-wifi-efi/firmware.sh get_from_efi
-    sudo umount /tmp/apple-wifi-efi
-    ```
-
-3. Setup `iwd` as backend for NetworkManager. You can run the following commands in Konsole to do so:
-    ```sh
-    cat <<EOF | sudo tee /etc/NetworkManager/conf.d/wifi_backend.conf
-    [device]
-    wifi.backend=iwd
-    EOF
-    sudo systemctl restart NetworkManager
-    ```
+It is highly recommended to create a bootable macOS installer **before** you modify your partitions or install CachyOS, especially if you are considering removing macOS entirely. This USB installer allows you to reinstall macOS for firmware updates, troubleshooting, or restoring your system if needed.
```

Comment:
This section looks like a compulsory step rather than a recommendation. Can we format it in such a way that it is separate from the main steps? 


A better way imo will be to just link Apple's guide here and remove the steps for bootable macOS usb.